package.json

```diff
@@ -387,6 +392,10 @@
 				"view": "gitpod-login",
 				"when": "gitpod.authenticated != true && !gitpod.inGitpodFlexRemoteWindow",
 				"contents": "You have not yet signed in with Gitpod\n[Sign in](command:gitpod.signIn)"
+			},
+			{
+				"view": "gitpod-flex-help",
+				"contents": "Connecting to Gitpod Flex environments requires installing the [Gitpod Flex extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-flex) and opening them through the [Gitpod Flex](https://app.gitpod.io/). This view is only to manage [Gitpod Classic](https://gitpod.io/workspaces) workspaces."
```

Comment:
`This view is only to manage [Gitpod Classic](https://gitpod.io/workspaces) workspaces.` can we visually separate this sentence, i.e. like a new paragraph.